test/e2e/webhelp.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { expect, test } from '@playwright/test';
+import { testSelector } from '../utils';
+import { WebHelpPage } from '../page/webhelp-page';
+import {  RESOLUTIONS } from '../visual/visual-constants';
+
+test.describe('WebHelp page interactivity', async () => {
+    test.beforeEach(async ({ page }) => {
+        const webHelpPage = new WebHelpPage(page, '/docs/test-page.html');
+        await webHelpPage.init();
```

Comment:
There is actually not so clear why we use class for just one method.

Reply:
I did so to maintain consistency in style between different tests but I agree that it may look like an overkill here